src/bindings/nodejs/package.json

```diff
@@ -14,6 +14,15 @@
       "types": "./index.d.ts"
     }
   },
+  "files": [
```

Comment:
can `protobufjs` be an optional dependency somehow? I'm not very familiar with nodejs. if not, then we should probably add it as a real dependency

Reply:
of course youre right! for some reason I thought we conditionally use probobufjs now in the library, but of course we dont.